pytest_mypy/item.py

```diff
@@ -163,6 +167,9 @@ def typecheck_in_new_subprocess(self, execution_path: Path, mypy_cmd_options: Li
 
         # add current directory to path
         self.environment_variables['PYTHONPATH'] = str(execution_path)
+        # Windows requires this to be set, otherwise the interpreter crashes
+        if 'SYSTEMROOT' in os.environ:
+            self.environment_variables['SYSTEMROOT'] = os.environ['SYSTEMROOT']
```

Comment:
FYI, just found out this is documented in [subprocess.Popen](https://docs.python.org/3/library/subprocess.html#subprocess.Popen)